set.mm

```diff
@@ -425366,6 +425495,35 @@ Norman Megill (2007) section 1.1.3.  Megill then states, "A number of
       WO $.
   $}
 
+  ${
+    $d x y A $.  $d x y B $.  $d x y F $.  $d x y G $.
```

Comment:
The comment added above says _Class and wff variables should appear at the beginning of distinct variable conditions_, it looks like this distinct variable condition (in the same PR!) does not adhere to it!

Reply:
> The comment added above says _Class and wff variables should appear at the beginning of distinct variable conditions_, it looks like this distinct variable condition (in the same PR!) does not adhere to it!

You are right, I just copied the distinct variable conditions from ~fpar. I will change it for ~ex-fpar.

Reply:
Done.